UnrealPlugin/README.md

```diff
@@ -47,4 +123,52 @@ public class <projectname>ServerTarget : TargetRules
 	}
 }
 
-```
\ No newline at end of file
+```
+
+Update DefaultGame.ini to show the following:
+ [/Script/UnrealEd.ProjectPackagingSettings]
+IncludeAppLocalPrerequesites=True
```

Comment:
Something I forgot to add that this can also be set in the UI in the editor. In the editor go to Edit -> Project Settings. In the opened window navigate to Packaging on the left side. Scroll to the bottom of the list, and tick "Include app-local prerequisities"